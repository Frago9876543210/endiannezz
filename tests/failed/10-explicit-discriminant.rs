use endiannezz::Io;

#[derive(Io)]
#[endian(big)]
#[repr(u8)]
enum Foo {
    Bar = 0xde,
    Baz,
}

fn main() {}
