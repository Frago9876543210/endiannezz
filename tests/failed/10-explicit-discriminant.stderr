error: All enum variants must have explicit discriminants
 --> $DIR/10-explicit-discriminant.rs:8:5
  |
8 |     Baz,
  |     ^^^
